core/src/main/scala/cats/Applicative.scala

```diff
@@ -67,6 +67,32 @@ import scala.annotation.implicitNotFound
   def replicateA[A](n: Int, fa: F[A]): F[List[A]] =
     Traverse[List].sequence(List.fill(n)(fa))(this)
 
+  /**
+   * Given `fa` and `n`, apply `fa` `n` times but discard the results.
+   *
+   * Example:
+   * {{{
+   * scala> import cats.data.State
+   *
+   * scala> type Counter[A] = State[Int, A]
+   * scala> val increment: Counter[Unit] = State.modify(_ + 1)
+   * scala> val get: Counter[Int] = State.get
+   * scala> val increment5AndGet: Counter[Int] = Applicative[Counter].productR(
+   *      | Applicative[Counter].replicateA_(5, increment),
+   *      | get
+   *      | )
+   * scala> increment5AndGet.run(0).value
+   * res0: (Int, Int) = (5,5)
+   * }}}
+   */
+  def replicateA_[A](n: Int, fa: F[A]): F[Unit] =
+    def go(x: Int, step: F[Unit]): F[Unit] = {
+      if(x == 0) step
+      else go(x - 1, this.productR(fa, step))
+    }
+    go(n, this.pure(()))
+  }
```

Comment:
Stray closing brace:

```suggestion
```

Reply:
Ah, I'm missing an opening brace. I don't know how I didn't catch that.